l10n_it_corrispettivi/README.rst

```diff
@@ -0,0 +1,122 @@
+====================================
+Italian Localization - Corrispettivi
+====================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--italy-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_corrispettivi
+    :alt: OCA/l10n-italy
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-italy-12-0/l10n-italy-12-0-l10n_it_corrispettivi
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/122/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+**Italiano**
+
+Questo modulo permette di generare le ricevute specificando le varie aliquote per riga
+(le righe della ricevuta hanno gli stessi automatismi di quelle della fattura) e quindi registrare le ricevute una per una.
+Un esempio tipico di questo caso d’uso è la vendita tramite sito di e-commerce.
+
+Chi invece emette scontrini (e non ha un POS integrato con Odoo) dovrà registrare in contabilità, a fine giornata, gli incassi totali.
+
+**English**
+
+This module allows you to generate receipts by specifying the different taxes per line
+(the receipt lines have the same automatisms as those on the invoice) and then record the receipts one by one.
+A typical example of this use case is the sale via e-commerce site.
+
+On the other hand, those who issue receipts (and do not have an integrated POS with Odoo) will have to record the total revenue at the end of the day.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+**Italiano**
+
+Creare almeno un registro di tipo vendita su cui saranno registrati i corrispettivi,
+deve avere il flag corrispettivi abilitato.
+
+Se necessario, creare una posizione fiscale per i corrispettivi (specificando l'azienda), deve avere il flag corrispettivi abilitato.
+Questa posizione fiscale verrà assegnata a tutti i nuovi partner (nell'azienda specificata) aventi il flag *Usa corrispettivi* abilitato.
+
```

Comment:
Per chiarezza spezzerei le frasi, evitando anche l'uso di "flag".
Un esempio potrebbe essere:
Creare almeno un registro di tipo vendita nel quale verranno registrati i corrispettivi.
È necessario che tali registri abbiano la casella "Corrispettivi" abilitata.

Se necessario, creare una posizione fiscale per i corrispettivi specificando l'azienda.
Deve inoltre essere selezionata la relativa casella "Corrispettivi".
Questa posizione fiscale verrà assegnata a tutti i nuovi partner, nell'azienda specificata, che hanno la casella *Usare corrispettivi* abilitata. 

Domanda, nella posizione fiscale non trovo la possibilità di specificare l'azienda come indicato nella configurazione. Ho capito male il senso della frase?

Reply:
Fatto in https://github.com/OCA/l10n-italy/compare/7ae2ab20552ee2abbe3c123532f80b0f22ce1b01..dccbc1c0fba9b5294b35a17cdc448b5b1e7dcb83

Reply:
> Fatto in https://github.com/OCA/l10n-italy/compare/7ae2ab20552ee2abbe3c123532f80b0f22ce1b01..dccbc1c0fba9b5294b35a17cdc448b5b1e7dcb83

Non è per essere pignolo, ma sarebbe meglio mantenere il modo verbale del resto del testo (infinito) e non rivolgersi direttamente all'utente:
Notare che il campo *Azienda*....

oppure usare il sostantivo:
Nota: il campo *Azienda*....

Perdonami :disappointed_relieved: :sweat_smile:

Reply:
Dopo questo è :+1:

Reply:
@primes2h fai bene, se sapessi l'italiano bene come te sarei uguale se non peggio.

Reply:
@monen17 Più che altro è deformazione da traduttore. :sweat_smile: